package.json

```diff
@@ -86,7 +86,8 @@
   "repository": "sequitur/improv",
   "scripts": {
     "prepublish": "gulp prepublish",
-    "test": "gulp"
+    "test": "gulp",
+    "demo": "gulp demo"
```

Comment:
Running gulp from package.json decouples the user's global gulp version from the one required by the package.